pkg/v1/cli/command/core/root.go

```diff
@@ -60,6 +60,7 @@ func NewRootCmd() (*cobra.Command, error) {
 		versionCmd,
 		completionCmd,
 		configCmd,
+		contextCmd,
```

Comment:
Any reason why `login` is a plugin but `context` is added to the core?

Reply:
Was part of the other PR. Have removed those elements.